indigo_za/terms.py

```diff
@@ -30,6 +30,18 @@ def setup(self, doc):
         self.term_re = re.compile('|'.join(in_quotes + self.others))
         super().setup(doc)
 
+    def contains_definition_block_container(self, element):
+        # returns True for the first blockContainer that has 'definition' as one of its classes
+        for block_container in element.xpath('.//a:blockContainer[@class]', namespaces=self.nsmap):
+            if 'definition' in block_container.get('class', '').split():
+                return True
+
+    def may_contain_definitions(self, element):
+        base_may_contain = super().may_contain_definitions(element)
+        if not base_may_contain:
+            return self.contains_definition_block_container(element)
+        return base_may_contain
```

Comment:
it's cleaner to do this:

```python
return super().may_contain_definitions(element) or self.contains_definition_block_container(element)
```

Python's short-circuit boolean logic will mean that if the first value in `a or b` is True, then `b` is not evaluated at all since it doesn't matter what the value of `b` is.